hack/logsamples/otlp_audit_ovn.json

```diff
@@ -0,0 +1,103 @@
+{
+  "resourceLogs": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "openshift.cluster.uid",
+            "value": {
+              "stringValue": "d24a392b-dd87-4eb8-a1a9-7384423567e3"
+            }
+          },
+          {
+            "key": "openshift.log.source",
+            "value": {
+              "stringValue": "ovn"
+            }
+          },
+          {
+            "key": "openshift.log.type",
+            "value": {
+              "stringValue": "audit"
+            }
+          },
+          {
+            "key": "k8s.node.name",
+            "value": {
+              "stringValue": "ip-10-0-36-191.ec2.internal"
+            }
+          },
+          {
+            "key": "openshift.label.foo",
+            "value": {
+              "stringValue": "bar"
+            }
+          },
+          {
+            "key": "log_type",
+            "value": {
+              "stringValue": "audit"
+            }
+          },
+          {
+            "key": "log_source",
+            "value": {
+              "stringValue": "ovn"
+            }
+          },
+          {
+            "key": "openshift.cluster_id",
+            "value": {
+              "stringValue": "d24a392b-dd87-4eb8-a1a9-7384423567e3"
+            }
+          },
+          {
+            "key": "kubernetes.host",
+            "value": {
+              "stringValue": "ip-10-0-36-191.ec2.internal"
+            }
+          }
+        ]
+      },
+      "scopeLogs": [
+        {
+          "scope": {
+
+          },
+          "logRecords": [
+            {
+              "timeUnixNano": "1743529186060055281",
+              "observedTimeUnixNano": "1743529186061086933",
+              "severityText": "INFO",
```

Comment:
Does this match the case of other samples?  Is there anything in the spec that should make this lowercase?

Reply:
yes, you are right